notes/content/00_setup/git-merge-driver.md

```diff
@@ -0,0 +1,23 @@
++++
+date = "2016-05-18T13:21:43+02:00"
+prev = "/00_setup/"
+next = "/01_upgrades/"
+title = "Git MergeDriver"
+weight = 10
++++
+
+Due to the dual use of monticello and git, there are some problems with merging.
+Git doesn't know how to merge monticello metadata files, causing a lot of merge conflicts.
+The manual solution for this is manually merging in Smalltalk first, and then committing the merged changes in git.
+
+This process can be automated by using the [filetree merge driver](https://github.com/ThierryGoubier/GitFileTree-MergeDriver).
+Unfortunately, it probably requires Linux or OSX (the merge script itself is a bash script).
+The setup is quite simple:
+
+ * Clone the git repository of the merge driver somewhere where it can stay (for example `/usr/local/`)
```

Comment:
I chose `/usr/local/` because I thought it would be the usual place for self compiled software. Its probably best just remove the part in the parentheses, it doesn't really matters after all.